src/components/Board.jsx

```diff
@@ -0,0 +1,102 @@
+import "./Board.css";
+import Square from "./Square";
+import { useState } from "react";
+
+export default function Board({ isX }) {
+  const [xIsNext, setXIsNext] = useState(isX);
+  const [squares, setSquares] = useState(Array(9).fill(null));
+
+  function handleClick(i) {
+    if (calculateWinner(squares) || squares[i]) {
+      return;
+    }
+    const nextSquares = squares.slice();
+    if (xIsNext) {
+      nextSquares[i] = "X";
+    } else {
+      nextSquares[i] = "O";
+    }
+    setSquares(nextSquares);
+    setXIsNext(!xIsNext);
+  }
+
+  const winner = calculateWinner(squares);
+  let status;
+  if (winner) {
+    status = "Winner: " + winner;
+  } else {
+    status = "Next player is: " + (xIsNext ? "X" : "O");
+  }
+
+  return (
+    <>
+      <div className="board">
+        <div className="status">{status}</div>
+        <div className="gamezone">
+          <div className="board-row">
+            <Square
+              value={squares[0]}
+              onSquareClick={() => handleClick(0)}
+            ></Square>
```

Comment:
Ինքնափակվող գրելաձևը ստեղ ավելի հարմար է՝

```suggestion
            />
```